src/Lean/Declaration.lean

```diff
@@ -226,10 +236,14 @@ def mkInductiveValEx (name : Name) (levelParams : List Name) (type : Expr) (numP
 def InductiveVal.numCtors (v : InductiveVal) : Nat := v.ctors.length
 
 structure ConstructorVal extends ConstantVal where
-  induct  : Name    -- Inductive Type this Constructor is a member of
-  cidx    : Nat     -- Constructor index (i.e., Position in the inductive declaration)
-  numParams : Nat   -- Number of parameters in inductive datatype `induct`
-  numFields : Nat   -- Number of fields (i.e., arity - nparams)
+  /-- Inductive Type this Constructor is a member of -/
```

Comment:
```suggestion
  /-- Inductive type this constructor is a member of -/
```
Any time you see weird casing, it's probably from our semi-automated naming style change :)